tests/expressions/tests.py

```diff
@@ -1351,6 +1355,12 @@ def test_expressions(self):
         self.assertEqual(repr(RawSQL('table.col', [])), "RawSQL(table.col, [])")
         self.assertEqual(repr(Ref('sum_cost', Sum('cost'))), "Ref(sum_cost, Sum(F(cost)))")
         self.assertEqual(repr(Value(1)), "Value(1)")
+        self.assertEqual(
+            repr(ExpressionList(F('col'), F('anothercol'))),
+            '<ExpressionList: F(col), F(anothercol)>')
+        self.assertEqual(
+            repr(ExpressionList(OrderBy(F('col'), descending=False))),
+            '<ExpressionList: OrderBy(F(col), descending=False)>')
```

Comment:
Please use hanging indentation as described in [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style)